frontend/src/components/GoalsTable/GoalRow.js

```diff
@@ -203,24 +213,42 @@ function GoalRow({
 
   const displayGoalTopics = truncateGoalTopics(goalTopics);
 
+  const closeOrOpenObjectives = (collapseFromObjectives) => {
+    if (collapseFromObjectives && expandObjectivesRef.current) {
+      expandObjectivesRef.current.focus();
+    }
+
+    setObjectivesExpanded(!objectivesExpanded);
+  };
+
+  const getStatusColor = () => {
+    if (goalStatus) {
+      const goalStatusDisplay = mapToDisplay.find((m) => m.stored === goalStatus);
+      if (goalStatusDisplay) {
+        return goalStatusDisplay.color;
+      }
+    }
+    return '#c5c5c5';
+  };
+
   return (
-    <tr className="tta-smarthub--goal-row" key={`goal_row_${id}`}>
-      <td>
-        {getGoalStatusIcon()}
-        {displayStatus === 'Ceased/suspended' ? ['Ceased/ ', <br />, 'suspended'] : displayStatus}
-      </td>
-      <td>{moment(createdOn).format(DATE_DISPLAY_FORMAT)}</td>
-      <td className="text-wrap maxw-mobile">
-        {goalText}
-        {' '}
-        (
-        {goalNumber}
-        )
-        {determineFlagStatus()}
-      </td>
-      <td className="text-wrap maxw-mobile">
-        {
-          showToolTip
+    <>
+      <tr className={`tta-smarthub--goal-row ${!objectivesExpanded ? 'tta-smarthub--goal-row-collapsed' : ''}`} key={`goal_row_${id}`}>
+        <td style={{ borderLeft: objectivesExpanded ? `4px solid ${getStatusColor()}` : '' }}>
+          {getGoalStatusIcon()}
+          {displayStatus}
+        </td>
+        <td>{moment(createdOn).format(DATE_DISPLAY_FORMAT)}</td>
+        <td className="text-wrap maxw-mobile">
+          {goalText}
+          {' '}
+          (
+          {goalNumber}
+          )
+          {determineFlagStatus()}
+        </td>
+        <td className="text-wrap maxw-mobile">
+          {showToolTip
             ? (
               <Tooltip
```

Comment:
The text needs to be read off, probably the prop names on this component are unclear

```suggestion
                      <Tooltip
                displayText={displayGoalTopics}
                screenReadDisplayText={false}
                buttonLabel={`click to visually reveal ${goalTopics.join(', ')}`}
                tooltipText={goalTopics.join(', ')}
                hideUnderline={false}
                svgLineTo={300}
              />
```